### ISSUE TITLE
Header refresher stores data globally

### ISSUE THREAD

**Issue by olloff** [revilheart/ESGST#398]
This is a minor issue, it won't block Header refresher functional.  
Version: v6.Beta.34.14 under Greasemonkey.

When 2 tabs with SG are opened, Header refresher updates every tab with data gathered from one. This leads to displaying incorrect amount of points, notifications, etc. Issue present for either container-tabs and normal-tabs.   
For example, giving SG runs on tabs A and B, nevertheless of using one set of credentials or different credentials for each tab: if some points was spent on tab A, tab B will be updated with amount of point left on tab A and vice-versa.

**Comment by revilheart** [comment]
That's normal behavior. To prevent each tab from making unnecessary requests, only one of the open tabs makes the requests and updates it to the other tabs. As a result, the data can indeed be outdated sometimes, but it should correct itself after 1 minute.

I believe there is no better way to handle this, because let's say a user has 10 SG tabs open. If each tab requested individually, there would be 10 requests being made to SG every single minute, leading to a lot of load on the servers. Instead, only one of the 10 tabs requests the data and sends it to the other tabs, making it only 1 request per minute.

**Comment by olloff** [comment]
Oh, ok. 
Can you ever consider adding a setting for this? Something about longer update timeout and disabling global updates.

**Comment by revilheart** [comment]
I can definitely add an option to allow users to specify the timeout duration, but I don't see the point of adding an option to disable global updates. It's still better to have only one of the open tabs do the refreshes and let it send the data to the other tabs. Allowing each tab to refresh individually would lead to unnecessary requests no matter how long the timeout is.

**Comment by olloff** [comment]
@revilheart I see your point. 
Actually I run two instances of SG in two containers, and when one instance updating, so other are. This causes wacky situations like mistaken notifications so I'm trying to find the way to suppress updating between containers. 
This is not so irritating when using one instance of SG.

**Comment by revilheart** [comment]
But how could there be mistaken notifications? They are for the same account, right?

**Comment by olloff** [comment]
No, they're not. So when one instance refreshed, it updates both tabs. Its annoying, but nothing more.
![giphy](https://user-images.githubusercontent.com/441860/30036259-3270a1fe-91ba-11e7-9597-28279db409f3.gif)

**Comment by revilheart** [comment]
Well, multiple accounts is against SG rules.

**Comment by olloff** [comment]
@revilheart I was actually. Though its not mine, we're sharing some sort of steambox with my brother and its easy to use with one browser window, but became pain in the ass with more so I adore every enhancement which can reduce amount of my clicks.
As I said, using one instance of SG in multiple tabs only cause mistakes in points count so its usable.
